packages/amplify-data-construct/.jsii

```diff
@@ -3991,7 +3991,7 @@
     "stability": "stable"
   },
   "homepage": "https://github.com/aws-amplify/amplify-category-api.git",
-  "jsiiVersion": "5.5.4 (build 1378d94)",
+  "jsiiVersion": "5.6.0 (build 7be6ace)",
```

Comment:
Why is the JSII version changing?

Reply:
I merged changes from main, then I ran `yarn build` and `yarn test`. That change appears after running `yarn build`.

Reply:
It probably shouldn't be changing from modifying the husky version.

Reply:
lemme try again